preprocessors/ner/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:20.04
+
+ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64/
+
+RUN apt-get update -y \
+        && apt-get install -y software-properties-common \
+        && add-apt-repository ppa:deadsnakes/ppa \
+        && apt-get install openjdk-8-jdk -y \
+        && apt-get install python3-pip -y \
+        && export JAVA_HOME \
+        && apt-get clean \
+        && rm -rf /var/lib/apt/lists/* \
+        && apt install python3-pip
+
+WORKDIR /app
+RUN adduser --disabled-password python
+ENV PATH="/home/python/.local/bin:${PATH}"
+
+COPY /schemas /app/schemas
+COPY /preprocessors/ner/ /app
+
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y python3-opencv gcc wget git && rm -rf /var/lib/apt/lists/*
```

Comment:
To confirm, are you currently using opencv/gcc/wget? IIRC you *are* using git.

Reply:
I'm using gcc and git, not opencv and get. I removed those.